tokenomics-explainers/002-rework-intro.md

```diff
@@ -44,10 +49,21 @@ Node-level penalties and forced validator exits are also included in the rework
 
 Megapools will enable multiple validators to share the same Ethereum withdrawal address – this **reduces initial and ongoing gas costs** for node operators.
 
-**RPL inflation will be reduced from 5% to 1.5%**, because the current 3.5% inflation used for RPL rewards to NOs is no longer necessary.
+**RPL issuance will be reduced from 5% to 1.5%**, because the current 3.5% inflation used for RPL rewards to NOs is no longer necessary.
 
 Forced delegate upgrades (with a grace period) will limit the technical debt the Rocket Pool protocol accumulates from previous versions.
 
+## Personas
+
+|Persona|Now|With Rework|
+|-|-|-|
+|**Existing Rocket Pool Node Operator**|I need RPL to create minipools. I have to remain collateralized at 10% of RPL/ETH to receive RPL rewards. My commission is 14% of borrowed ETH.|I do not need RPL to create minipools. I will receive node operator commission paid in ETH (3.5%). If I *do* stake RPL I will also receive voter share rewards (5.0%) paid in ETH (with no 10% minimum or cliff). I will either benefit from surplus share value accrual to RPL (5.5%) or receive an additional (5.5%) of voter share rewards. I will no longer receive RPL rewards.|
```

Comment:
```suggestion
|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL (which makes it vote-eligible) I will receive additional ETH rewards from voter share (5.0% of protocol revenue from all megapools) proportional to my vote-eligible RPL. I do not have a minimum staked RPL requirement (or "cliff") to maintain eligibility for voter share rewards, but my max vote eligibility is an RPL staked value of 150% of my bonded ETH. I will also benefit from either surplus share value accrual to RPL (5.5% of protocol revenue from all megapools) or a 5.5% increase to voter share. I will no longer receive RPL issuance
```

Reply:
Going to take this and potentially cut it down. Getting worried about length.